src/summary/index.js

```diff
@@ -501,6 +501,11 @@ function load_the_rest(storage) {
     document.getElementById("rowWeeks").style.display = "block";
     document.getElementById("rowDays").style.display = "block";
     document.getElementById("rowPrefs").style.display = "block";
+
+    var dataStr = "data:text/json;charset=utf-8," + encodeURIComponent(JSON.stringify(storage.days));
+    var dlAnchorElem = document.getElementById('downloadAnchorElem');
+    dlAnchorElem.setAttribute("href", dataStr);
+    dlAnchorElem.setAttribute("download", "scene.json");
```

Comment:
For the file name, instead of "scene.json" how about "mind-the-time.json"?  For extra polish it would be nice to include the date in the file name like "mind-the-time-2020-06-07.json", but that's only if you want to add that as part of this change, can always be added later.